lms/djangoapps/discussion/rest_api/views.py

```diff
@@ -90,7 +90,8 @@ class CourseView(DeveloperErrorViewMixin, APIView):
             * end: The ISO 8601 timestamp for the end of the blackout period
 
         * thread_list_url: The URL of the list of all threads in the course.
-
+            # Question for Kshitij (Remove before merging)
```

Comment:
@xitij2000 Please let me know about this one

Reply:
Yes.